### PR TITLE
add dependencies for PR: #5698

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
Add missing libs dependencies for PR https://github.com/mattermost/platform/pull/5698

### CODE REVIEW

**Comment by jasonblais** [comment]
@crspeller It looks like #6215 removed the following library

-- package: gopkg.in/gomail.v2
 -  version: 2.0.0

and in Carlos' also had this in his `glide.lock` as well

- name: github.com/dimchansky/utfbom
  version: 6c6132ff69f0f6c088739067407b5d32c52e1d0f

@crspeller Can you help with review?

**Comment by jasonblais** [comment]
These PR is for https://github.com/mattermost/platform/pull/5698

File: glide.lock [hunk 1]
```diff
@@ -12,6 +12,8 @@ imports:
   version: d788b7d6dc58bea298b318059332a25442583880
 - name: github.com/dgryski/dgoogauth
   version: 96977cbd42e27be71f9f731db6634123de7e861a
+- name: github.com/dimchansky/utfbom
```
Comment (hmhealey) [inline glide.lock]: Do we still need utfbom? I can't see anything in your other PR that requires it

Comment (cpanato) [inline glide.lock]: @hmhealey the github.com/cpanato/html2text uses that.

Comment (hmhealey) [inline glide.lock]: Ah knew I missed something important

**Comment by cpanato** [comment]
no longer needed due this https://github.com/mattermost/platform/pull/6431

**Pull request closed without merging**